rest-api-spec/src/main/resources/rest-api-spec/api/cat.segment_replication.json

```diff
@@ -0,0 +1,108 @@
+{
+  "cat.segment_replication":{
+    "documentation":{
+      "url":"https://github.com/opensearch-project/documentation-website/issues/2627",
+      "description":"Returns information about both on-going and latest completed Segment Replication events"
+    },
+    "stability":"stable",
```

Comment:
@Rishikesh1159  Should probably make this `experimental`, no?

Reply:
Thanks @andrross for catching this. Yes I missed this, it should be `experimental`. Let me make a PR to change this.